concrete/src/Multilingual/Service/Extractor.php

```diff
@@ -72,6 +77,8 @@ public function extractTranslatableSiteStrings()
 
     public function getDynamicTranslations()
     {
+        $fh = \Core::make('helper/file');
+        \C5TL\Options::setTemporaryDirectory($fh->getTemporaryDirectory());
```

Comment:
Also lets add use statements for these to match what we currently have in: https://github.com/concrete5/concrete5/issues/3539